docs/dev/Installation_and_setup.md

```diff
@@ -0,0 +1,152 @@
+
+  
+## Driver Installation
+
+The driver can be obtained by downloading the bundled distribution installer from [here](https://opendistro.github.io/for-elasticsearch/downloads.html) or by building from source.
+
+###  Via Installer
+
+#### Windows
+
+1. Double click the downloaded `ODFE SQL ODBC Driver-<version>-Windows.msi` installer. In the case installer is unsigned it will show the following pop up. Click on **More info** and click on **Run anyway**. 
+Then click **Next** to proceed installation.
+
+<p align="center">
+  <img src="img/windows_signing_error_1.png" width="400" height="400">
+  <img src="img/windows_singing_error_2.png" width="400" height="400">
+</p>
+
+<p align="center">
+  <img src="img/windows_installer_home.png" width="506" height="397">
+</p>
+
+
+2. Accept the agreement and click **Next**.
+
+3. The installer comes bundled with documentation and useful resources files to connect with various BI tools, for example `.tdc` file for Tableau. You can choose to keep the documentation and resources, or remove it. You can also choose the download location. Click **Next**.
+
+<p align="center">
+  <img src="img/windows_installer_select_and_browse.png" width="506" height="397">
+</p>
+
+
+4. Click **Install** and then **Finish**.
+
+<p align="center">
+  <img src="img/windows_installer_install.png" width="506" height="397">
+</p>
+
+
+5. The **DSN** come already setup with the installation. The following connection information is setup as part of default DSN
+
+//TODO; add config
+
+----
+#### MacOS
+
+1. Double click the downloaded `ODFE SQL ODBC Driver-<version>-Darwin.pkg` installer. In the case installer is unsigned it will show the following pop up. Right click on installer and select **Open**. 
+Click **Continue** to proceed installation.
+
+<p align="center">
+  <img src="img/mac_signing_error_1.png" width="417" height="211">
+  <img src="img/mac_signing_error_2.png" width="417" height="211">
+</p>
+
+
+<p align="center">
+  <img src="img/mac_installer_home.png" width="506" height="397">
+</p>
+
+2. Click **Continue** to move past the **Introduction** and **Readme** .
+
+3. Accept the agreement and click **Continue**.
+
+<p align="center">
+  <img src="img/mac_installer_license.png" width="506" height="397">
+</p>
+
+
+4. Choose **Destination** to install driver files.
+
+5. The installer comes bundled with documentation and useful resources files to connect with various BI tools, for example `.tdc` file for Tableau. You can choose to keep the documentation and resources, or remove it. Click **Customize** to choose needed files. Click **Continue**.
+
+<p align="center">
+  <img src="img/mac_installer_select_and_browse.png" width="506" height="397">
+</p>
+
+4. Click **Install** and then **Close**.
+
+<p align="center">
+  <img src="img/mac_installer_succesful.png" width="506" height="397">
+</p>
+
+5. Currently the **DSN** is not setup as part of installation and need to be configured manually.
```

Comment:
Can link to our README or copy those instructions here (this section or somewhere else) to make it clear how to configure a DSN on Mac.